Source/cmLinkDatabaseGenerator.cpp

```diff
@@ -0,0 +1,62 @@
+//
+// Created by Gleb Mirzazyanov on 31.05.2021.
```

Comment:
Please remove this comment. You need to add license. See it in other files

<!--StartFragment-->/* Distributed under the OSI-approved BSD 3-Clause License.  See accompanying   file Copyright.txt or https://cmake.org/licensing for details.  */<!--EndFragment-->